pullpigo.go

```diff
@@ -19,13 +19,41 @@ func main() {
 	println()
 	events := githubEvents(config)
 
-	eventsByAuthor := make(map[actor]int)
+	pullRequestCreatedByAuthor := make(map[actor][]rawEvent)
+	pullRequestClosedByAuthor := make(map[actor][]rawEvent)
+	pullRequestCommentedByAuthor := make(map[actor][]rawEvent)
+
 	for _, event := range events {
-		eventsByAuthor[event.Actor]++
+		if event.pullRequestCreated() {
+			pullRequestCreatedByAuthor[event.Actor] = append(pullRequestCreatedByAuthor[event.Actor], event)
+		}
+		if event.pullRequestClosed() {
+			pullRequestClosedByAuthor[event.Actor] = append(pullRequestClosedByAuthor[event.Actor], event)
+		}
+		if event.pullRequestComment() {
+			pullRequestCommentedByAuthor[event.Actor] = append(pullRequestCommentedByAuthor[event.Actor], event)
+		}
+	}
+	if len(pullRequestCreatedByAuthor) > 0 {
+		println("pull requests created")
+		for author, events := range pullRequestCreatedByAuthor {
+			fmt.Printf("  %v by %s", len(events), author.Login)
```

Comment:
Update [the output sample in the README](https://github.com/nicokosi/pullpigo#run). :innocent:

Reply:
Make the output similar with https://github.com/nicokosi/pullpito/#run ?